talk/owt/sdk/include/cpp/owt/base/stream.h

```diff
@@ -43,6 +43,8 @@ class StreamObserver {
   virtual void OnEnded() {};
   /// Triggered when the stream info is updated in conference mode.
   virtual void OnUpdated() {};
+  virtual void OnMute(TrackKind track_kind) {};
```

Comment:
Please add API description for newly added public APIs. Also prefer to use `const TrackKind&`.

Reply:
@taste1981 @HuaChunbo Please take a look. We're using `TrackKind` in publication and subscription. Would you prefer to change all of them to `const TrackKind&` or keep using `TrackKind` here?

Reply:
Suggest to just use TrackKind here since it is an enum type here.